code/game/machinery/doors/table.dm

```diff
@@ -0,0 +1,293 @@
+//Its a door. Doors are too machiney, lets make it a structure instead! -Oldcoders
+//That was a bad idea, lets make it a machine instead!
+
+/obj/machinery/door/table
+	name = "table door"
+	opacity = 0
+	animation_delay = 0
+	pass_flags_self = PASSTABLE
+	layer = TABLE_LAYER
+	open_layer = TABLE_LAYER
+	throwpass = 1	//You can throw objects over this, despite its density.
+	use_power = MACHINE_POWER_USE_NONE
+	machine_flags = SCREWTOGGLE
+	icon = 'icons/obj/doors/tabledoor.dmi'
+	icon_state = "metaldoor_closed"
+	prefix = "metal" //Corresponds to the mineral type
+	health = 100
+	soundeffect = 'sound/effects/wood_door_slam.ogg'
+	var/obj/item/weapon/circuitboard/airlock/electronics = null
```

Comment:
I wonder if it wouldn't make more sense for this to be in `contents` but idk i'll trust this implementation